handler/src/main/java/io/netty/handler/ssl/SniHandler.java

```diff
@@ -56,7 +58,7 @@
             InternalLoggerFactory.getInstance(SniHandler.class);
     private static final Selection EMPTY_SELECTION = new Selection(null, null);
 
-    protected final AsyncMapping<String, SslContext> mapping;
+    protected final SslContextMapping mapping;
```

Comment:
@rkapsi yes as @Scottmitch noted this is a breaking change so sorry but no we can not do this.